admin_organization_test.go

```diff
@@ -22,8 +22,10 @@ func TestAdminOrganizations_List(t *testing.T) {
 		adminOrgList, err := client.Admin.Organizations.List(ctx, AdminOrganizationListOptions{})
 		require.NoError(t, err)
 
-		assert.Equal(t, true, adminOrgItemsContainsName(adminOrgList.Items, org.Name))
-		assert.Equal(t, 1, adminOrgList.CurrentPage)
+		// Given that org creation occurs on every test, the ordering is not
+		// gauranteed. It may be that the `org` created in this test does not appear
```

Comment:
Ah nice, good catch.

```suggestion
		// guaranteed. It may be that the `org` created in this test does not appear
```